lib/ast-converter.js

```diff
@@ -420,6 +420,8 @@ module.exports = function(ast, extra) {
             return null;
         }
 
+        var accessibility;
```

Comment:
I really don't like this line. I'm going to have it always set the `accessibility` and just have it default to `null` instead to clean up code.